barema4/React-challenge
# Revert "#163810557 create empty react project"

### What does this PR do?
Sets up an empty React project

### Description of the task to be completed?
- Add .gitignore, README.md, and LICENSE.txt
- Initialize repository as a yarn project (adds package.json)
- Add dependencies for Webpack, Babel and React to package.json
- Create .babelrc to configure Babel
- Add Webpack configuration to webpack.config.js
- Create basic application files (index.html, index.js, App.js)
- Setup ESLint using Airbnb rules

### How should this be manually tested?
- Install all dependencies using `yarn`
- Run the server `yarn start` t
- Open `localhost:3000` in a browser to verify that the app is running
- To lint, use the command: `yarn run lint`
### What are the relevant pivotal tracker stories?
[#163810557](https://www.pivotaltracker.com/story/show/163810557)
